Source/ACE/Entity/Player.cs

```diff
@@ -728,9 +894,9 @@ public void Logout(bool clientSessionTerminatedAbruptly = false)
 
             // NOTE: Adding this here for now because some chracter options do not trigger the GameActionSetCharacterOptions packet to fire when apply is clicked (which is where we are currently saving to the db).
             // Once we get a CharacterSave method, we might consider removing this and putting it in that method instead.
-            DatabaseManager.Character.SaveCharacterOptions(character);
+            //DatabaseManager.Character.SaveCharacterOptions(character);

```

Comment:
You commented this out, but why?  If it's implemented elsewhere, just delete it.  if not, it'd be good to know why this is no longer needed.